packages/ckeditor5-list/docs/_snippets/features/lists-document.html

```diff
@@ -6,39 +6,39 @@ <h3>...and stay on the budget.</h3>
 			<h4>Gran Canaria, Spain</h4>
 			<figure class="image image_resized image-style-align-right" style="width:30%;">
 				<img src="%BASE_PATH%/assets/img/gran-canaria.jpg">
-				<figcaption>Some view of Gran Canaria</figcaption>
+				<figcaption>Ocean shore of Gran Canaria</figcaption>
 			</figure>
-			<p>Gran Canaria is a popular tourist destination. The island is often called "a miniature continent". This is owed to the fact, that is offers a great variety of climates and landscapes, from golden, sandy beaches, through rocky ravines, white dunes and charming villages. In the mid-2010s over 3.5 million tourists visited Gran Canaria annually.</p>
+			<p>Gran Canaria is a popular tourist destination. The island is often called "a miniature continent". This is owed to the fact, that it offers a great variety of climates and landscapes, from golden, sandy beaches, through rocky ravines, white dunes, and charming villages. In the mid-2010s over 3.5 million tourists visited Gran Canaria annually.</p>
 		</li>
 		<li>
-			<h4>Florence, Italy</h4>
-			<p>Florence, established in 59BC by Julius Caesar himself got its name from two river that surrounded the settlement. Thanks to centuries of history, the capital of Tuscany has plenty to offer for tourists hungry of sightseeing. Some of the most interesting places to visit include:</p>
+			<p>Florence, Italy</p>
```

Comment:
Why is this a paragraph now?

![](https://user-images.githubusercontent.com/1099479/168542394-8461ca6c-e210-441d-8fea-1d30c46a8fcf.png)

Reply:
Not intentionally, must've changed during some paste. I'll restore it.